crates/forge_analyzer/src/definitions.rs

```diff
@@ -2981,11 +3503,42 @@ impl Environment {
             DefKind::Undefined => DefKind::Undefined,
         }
     }
+    pub fn is_import(&self, import_kind: ImportKind, named_import: Option<String>) -> bool {
+        // debug!("{named_import}");
+        match import_kind {
+            ImportKind::Star => true,
+            ImportKind::Default => true,
```

Comment:
This should be true iff `named_import` is "default", since this function is for checking if the import matches whatever identifier/string we pass in.